libs/couch2pg/importer.js

```diff
@@ -0,0 +1,105 @@
+var _ = require('underscore'),
+    Promise = require('rsvp').Promise,
+    format = require('pg-format');
+
+var deleteDocuments = function(db, docIdsToDelete) {
+  if (docIdsToDelete && docIdsToDelete.length > 0) {
+    return db.query(
+      format('DELETE FROM couchdb WHERE doc->>\'_id\' in (%L)',
+        docIdsToDelete));
+  } else {
+    return Promise.resolve();
+  }
+};
+
+var storeSeq = function(db, seq) {
+  return db.query(format('UPDATE couchdb_progress SET seq = %L', seq));
```

Comment:
To check my understanding:

So, in a possible future, we might mark certain CouchDB docs with an id like `_dbname/pgconfig`, then the adapter could import that specifically from CouchDB and place it into a `couchdb`-like table containing other JSON docs specifically for adapter/postgres config. The same doc would probably end up in the `couchdb` table as a matter of course, unless we put filters on, but it wouldn't get used from that location.

If I got it, yeah that sounds like a good future plan.